RealmSwift.xcodeproj/project.pbxproj

```diff
@@ -912,13 +892,8 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
-				FRAMEWORK_SEARCH_PATHS = (
-					build/ios,
-					../ios/dynamic,
-				);
-				GCC_PREPROCESSOR_DEFINITIONS = (
-					"$(inherited)",
-				);
+				FRAMEWORK_SEARCH_PATHS = "build/ios-dynamic";
+				GCC_PREPROCESSOR_DEFINITIONS = "$(inherited)";
```

Comment:
Why all these changes of `GCC_PREPROCESSOR_DEFINITIONS` to be strings instead of arrays?